fastlane/Appfile

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+json_key_file('') # Path to the json secret file - Follow https://docs.fastlane.tools/actions/supply/#setup to get one
+package_name('com.nimbl3') # your package name here
```

Comment:
@luongvo fixed

Reply:
Should we update `package_name` in the following PR as well ? https://github.com/nimblehq/android-templates/pull/44/files

Reply:
@Tuubz I will merge this PR first, then will rebase the package name to be correct in https://github.com/nimblehq/android-templates/pull/48 . Does it make sense? 🙏